mandatory/4-tax.js

```diff
@@ -17,7 +22,12 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(b) {
+  return "£" + ((20 / 100 * b) + b).toFixed(2);
+}
```

Comment:
Good use of .toFixed()

`(20 / 100 * b) + b` can be written as `b * 1.2` but your version is better when the percentage is being passed in as a value eg 20

Reply:
Hi Pete, I'm not sure if I understand what you mean. Can you please explain this more. Should I change it to b * 1.2?